### DISCUSSION TITLE
主应用是普通的vue项目（非umi搭建）， 子应用umi搭建。 问题：加载不出子应用

### DISCUSSION THREAD

**Discussion by morepowerful** [umijs/umi#12506] [General]
<img width="1725" alt="image" src="https://github.com/umijs/umi/assets/15519137/34deb89d-bc0d-478a-a57c-30065cfb7b48">

主应用代码如下（手动加载子应用）：
<img width="778" alt="image" src="https://github.com/umijs/umi/assets/15519137/9c219c3f-2a7b-4c09-aa80-2380c1bd6403">
<img width="595" alt="image" src="https://github.com/umijs/umi/assets/15519137/7975d43e-fc59-40e1-a690-2b4faca22dd8">

子应用主要代码如下：
<img width="493" alt="image" src="https://github.com/umijs/umi/assets/15519137/cb50933e-0b0e-4787-9fb6-5ae79fe6fc80">
<img width="676" alt="image" src="https://github.com/umijs/umi/assets/15519137/459093e2-ba05-4f37-bd08-922b0e4cf562">
<img width="785" alt="image" src="https://github.com/umijs/umi/assets/15519137/9775fc85-d97f-40bf-94da-16a38eb0ba6b">
<img width="844" alt="image" src="https://github.com/umijs/umi/assets/15519137/1a105d42-81ee-4c08-aa55-71ba6a9416ac">

demo: https://github.com/morepowerful/qiankun-umi-demo

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4 。
